aries-site/src/pages/templates/matrix-navigation.mdx

```diff
@@ -1,3 +1,19 @@
-import { ComingSoon } from '../../components';
+import { DiagramPreview, Example } from '../../layouts';
 
-<ComingSoon />
\ No newline at end of file
+## When a User would Use:
+- Pages where the hierarchy isn't crucial to the user experience, and where browsing is encouraged. For ex: HPE.com
+- User flows where a global nav bar is visible at all times.
+
+### Matrix within Global Header Example
+Global Header can provide easy access points to allow a user to jump from one section to another at all times. 
+
+<Example caption="Diagram illustrating the matrix navigation within a global header." plain pad="small" previewWidth="large" height={{ min: 'small' }}>
+<DiagramPreview src={{dark: "/templateImages/Matrix-Global-Header.svg", light: '/templateImages/Matrix-Global-Header-Invert.svg'}} alt="Diagram illustrating the matrix navigation within a global header." />
+</Example>
+
+### Matrix within App Example
+Users can search for information within a set of cards to select or navigate to different access points in no particualr order.
+
+<Example caption="Diagram illustrating the matrix navigation within an application." plain pad="small" previewWidth="large" height={{ min: 'small' }}>
+<DiagramPreview src={{dark: "/templateImages/matrix-app-example.svg", light: '/templateImages/matrix-app-example-invert.svg'}} alt="Diagram illustrating the matrix navigation within an application." />
```

Comment:
I noticed a bug in "DiagramPreview" where the "light" src is being applied in dark mode. I'm fixing this in my PR for "side-to-side" nav. But just to prep for that change, the `-invert` svg should be for the "dark" mode src

Reply:
thanks Ill fix that